src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String list[] = new String[100];
```

Comment:
Do you think it would be more beneficial to have a more descriptive name like "tasks"?

Reply:
That might be better! Thank you!